src/Initializer/CellLocalMatrices.cpp

```diff
@@ -164,6 +169,24 @@ void seissol::initializers::initializeCellLocalMatrices( MeshReader const&
         MeshTools::normalize(tangent1, tangent1);
         MeshTools::normalize(tangent2, tangent2);
 
+        real NLocalData[6*6];
+        seissol::model::getBondMatrix(normal, tangent1, tangent2, NLocalData);
+        if (material[cell].local.getMaterialType() == seissol::model::MaterialType::anisotropic) {
```

Comment:
You could also implement getRotatedMaterialCoefficients in seissol::model::Material (doing nothing for elastic/viscoelastic). Then you don't need the if here.